blackbox/docs/general/ddl/views.rst

```diff
@@ -45,7 +45,8 @@ Privileges (Enterprise only)
 In order to be able to query data from a view, a user needs to have ``DQL``
 privileges on a view.
 This is the case if the user has ``DQL`` privileges on a cluster level, or on
-the schema in which the view is contained.
+the schema in which the view is contained. Privileges on tables accessed by a
```

Comment:
Maybe rephrase this a bit..

This is the case if the user has ``DQL`` privileges on a cluster level, the schema in which the view is contained, or the view itself.

Maybe also connect the "Privileges on tables accessed by a view are not necessary" somehow better to the following paragraph that the owner needs to have permissions on the underlying relations. (I'd avoid limiting this to "tables", a view can contain other views)